arch/sim/src/sim/sim_head.c

```diff
@@ -113,8 +113,6 @@ const char *__asan_default_options(void)
          " alloc_dealloc_mismatch=0"
          " allocator_frees_and_returns_null_on_realloc_zero=0"
          " check_initialization_order=1"
-         " detect_invalid_pointer_pairs=2"
-         " detect_stack_use_after_return=1"
```

Comment:
but should we keep detect_stack_use_after_return which is wrong in the most case?

Reply:
stack check of asan will have a misjudgment. In the coroutine environment, asan always checks the host  stack  top/bottom.